### DISCUSSION TITLE
MapControl

### DISCUSSION THREAD

**Discussion by Phil4121** [CommunityToolkit/Maui#277] [New Feature Discussions]
Because a lot of Apps are using Maps, it would be great if a MapControl would be added into the Toolkit!
I've searched a lot but I could not find any project or implementation which can be easly implemented into a .Net Maui project...

**Comment by pictos** [comment]
@Phil4121 the Map control is on Xamarin.Forms and will be on .NET MAIU out of the box.

**Reply by Phil4121** [reply]
Unfortunately the implementation for the control has been removed from the Roadmap... I think it was planed for RC

@pictos may you have information about the timeline? Or could someone please post a sample on how to implement the MapControl from Xamarin? I've got serious problems implementing it.....

**Reply by pictos** [reply]
@Phil4121 I don't have any info about ETA. For Xamarin you can check the official docs [here](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/map/)

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
